extension/src/plots/model/util.ts

```diff
@@ -0,0 +1,20 @@
+import { getDataFromColumnPath } from '../../experiments/model/util'
+import { Experiment } from '../../experiments/webview/contract'
+
+export const getRevisionFirstThreeColumns = (
```

Comment:
In `experiments/model/tree.ts` `getTooltipTable` you have something very similar. Should these two function share some code?

Reply:
Looking at the two functions, they do similar things but I'm not seeing a way to share code since they take the results from `getDataFromColumn` and do different things to the returned values.